content/en/docs/reference/instrumentation/understand-psi-metrics.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Understand Pressure Stall Information (PSI) Metrics
+content_type: reference
+weight: 50
+description: >-
+  Detailed explanation of Pressure Stall Information (PSI) metrics and how to use them to identify resource pressure in Kubernetes.
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.34" state="beta" >}}
+
+As a beta feature, Kubernetes lets you configure the kubelet to collect Linux kernel
+[Pressure Stall Information](https://docs.kernel.org/accounting/psi.html)
+(PSI) for CPU, memory, and I/O usage. The information is collected at node, pod and container level.
+This feature is enabled by default by setting the `KubeletPSI` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+
+PSI metrics are exposed through two different sources:
+- The kubelet's [Summary API](/docs/reference/config-api/kubelet-stats.v1alpha1/), which provides PSI data at the node, pod, and container level.
+- The `/metrics/cadvisor` endpoint on the kubelet, which exposes PSI metrics in the [Prometheus format](/docs/concepts/cluster-administration/system-metrics#psi-metrics).
+
+### Requirements
+
+Pressure Stall Information requires:
+
+- [Linux kernel versions 4.20 or later](/docs/reference/node/kernel-version-requirements#requirements-psi).
```

Comment:
you also need to turn it on with a kernel arg I believe

Reply:
Added kernel config clarifications. It's not mentioned in the [kernel documentation](https://docs.kernel.org/accounting/psi.html), but I did some research and found some related configs. PTAL.